keel/api/v1/document/views.py

```diff
@@ -0,0 +1,46 @@
+from django.http import FileResponse
+
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from rest_framework import mixins, viewsets, status
+from rest_framework.response import Response
+
+from keel.document.models import Documents
+
+from .serializers import DocumentsSerializer
+
+
+
+class GetDocument(GenericViewSet):
+
+    def generate(self, request, format = 'json', **kwargs):
+        
+        response = {
+                "status": 0,
+                "message":"Document Fetched successfully",
+                "data": ""
+        }
+
+        doc_id = kwargs.get('doc_id')
```

Comment:
use request for every external entity..
because that the only source of data for us!!
here request.data will give you the payload...

Reply:
@KeelTech  Since doc_id is the path parameter for